ttt.py

```diff
@@ -101,6 +101,9 @@ def transcribe_fasterwhisper(calljson, audiofile):
         vad_filter=vad_filter,
         vad_parameters=dict(min_silence_duration_ms=500),
         language=language,
+        # This enhances distil models but is not required for "normal"
+        # Without it, distil models are bonkers.
+        condition_on_previous_text=False,
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **question (code_clarification):** Clarify the impact of disabling 'condition_on_previous_text'.

This change might affect the coherence of transcribed segments for certain audio inputs. It would be beneficial to document the scenarios where this setting improves performance.